src/api/java/blusunrize/immersiveengineering/api/IEProperties.java

```diff
@@ -196,8 +200,6 @@ public int hashCode()
 		public static final ModelProperty<Map<String, String>> TEXTURE_REMAP = new ModelProperty<>();
 		public static final ModelProperty<ConnectionModelData> CONNECTIONS = new ModelProperty<>();
 		public static final ModelProperty<MineralMix[]> MINERAL = new ModelProperty<>();
-		public static final ModelProperty<IConveyorBelt> CONVEYOR = new ModelProperty<>();
-		public static final ModelProperty<FeedthroughData> FEEDTHROUGH = new ModelProperty<>();
 		public static final ModelProperty<Map<Direction, IOSideConfig>> SIDECONFIG = new ModelProperty<>();
 		public static final ModelProperty<BlockPos> SUBMODEL_OFFSET = new ModelProperty<>();
 		//TODO remove?
```

Comment:
Well? Do we still need this? Either we remove the property or the TODO. Now would be the chance.

Reply:
I guess making it non-API would be an option. The problem is that on the one hand passing the complete TE defeats the point of model properties, is a threading nightmare and makes caching hard; while on the other hand a lot of code currently uses this functionality in ways that aren't trivial to change.